src/checkbox/checkbox.style.tsx

```diff
@@ -75,6 +63,28 @@ export const StyledInteremediateIcon = styled(MinusSquareFillIcon)<StyleProps>`
             : Colour["icon-selected"](props)};
 `;
 
+export const Container = styled.div<StyleProps>`
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    position: relative;
+
+    ${(props) => css`
+        height: ${props.$displaySize === "small" ? "1.5rem" : "2rem"};
+        width: ${props.$displaySize === "small" ? "1.5rem" : "2rem"};
+    `}
+
+    // Show custom focus ring when input is focused
+    input:focus-visible + ${StyledUncheckedIcon},
+    input:focus-visible + ${StyledUncheckedDisabledIcon},
+    input:focus-visible + ${StyledCheckedIcon},
+    input:focus-visible + ${StyledInteremediateIcon} {
```

Comment:
can simplify this to `input:focus-visible + svg` since these are all using the icon svgs